torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -317,6 +320,21 @@ def codegen_define(self, kernel: CppTemplateKernel) -> str:
         return KernelTemplate._template_from_string(self.TEMPLATE_ENTRY).render(options)
 
 
+# extra check for small M dimension for int8 WoQ case
+def check_int8_woq_small_m_dim(config, m, n, k, alpha, num_threads, **kwargs):
+    return (
+        k % config.register_blocking.block_k == 0
+        and n % config.register_blocking.block_n == 0
+        and m < 16
+        and not kwargs.get("dynamic_M", False)
+    )
+
+
+# For int8 WoQ GEMM with small M, we use different blockings that shouldn't be used otherwise
+def do_not_use_with_small_m(config, m, n, k, alpha, num_threads, **kwargs):
```

Comment:
Can we improve the naming of this function? It's not only about small m. It's also about woq int8.